simulationfactory/package-lock.json

```diff
@@ -0,0 +1,14227 @@
+{
```

Comment:
Can we add this to the .gitignore?

Reply:
yes. I googled around & for some reason, no one mentioned to add that file to gitignore, but it seems to contain machine-specific code

Reply:
If it's machine-specific, I suggest we add it to the .gitignore.  This file probably isn't big enough to cause any problems, but git doesn't play nice with big auto-generated files like this